### PR TITLE
More Navigation Added For Successful Signup Login & Club to Club Profile Navigation

### PR DESCRIPTION
**Pull request by ShengqiWu2004**

### CODE REVIEW

File: frontend/src/pages/LoginSignup/LoginCard.tsx [hunk 1]
```diff
@@ -114,6 +114,8 @@ const LoginCard: React.FC<LoginCardProps> = ({ accountType }) => {
             // Handle other error codes (401 unauthorized, etc)
             console.error('There was a problem with the fetch operation:', error);
         }
+        //SW ADDED
+        navigate('/events');//Please only put this in the success path
```
Comment (harry-lons) [inline frontend/src/pages/LoginSignup/LoginCard.tsx:118]: This navigation is already on line 107

Comment (ShengqiWu2004) [reply frontend/src/pages/LoginSignup/LoginCard.tsx:118]: That's true. I am sorry about this mistake.

File: frontend/src/pages/LoginSignup/SignupCard.tsx [hunk 2]
```diff
@@ -40,6 +40,8 @@ const SignupCard: React.FC<SignupCardProps> = ({ accountType }) => {
         formData.append('password', enteredPassword);
 
         // TODO: Send backend request to sign up (add the user to the database)
+        // SW ADDED:
+        navigate('/login');//Please only put this in the successful path
```
[the remark this answers is not in the document]
Comment (ShengqiWu2004) [reply frontend/src/pages/LoginSignup/SignupCard.tsx:44]: That's a great idea. I need to specify the paths

**Comment by ShengqiWu2004** [comment]
changes made accordingly in DetailedEvent AddEventForm and EditEventForm. New Util added for getting all RSVP for a certain event. Extra navigation in Login/Signup removed

**Pull request merged**